openage/convert/game_versions.py

```diff
@@ -80,22 +80,22 @@ class GameVersion(enum.Enum):
     # HD edition version 4.0
     age2_hd_fe = (
         12.0,
-        "Age of Empires 2: HD + Forgotten Empires (Version 4.0+)",
+        "Age of Empires 2: HD (Version 4.0+)",
         Support.yes,
         {'AoK HD.exe', 'resources/_common/dat/empires2_x1_p1.dat'},
```

Comment:
Hmm.. Not sure if this is the best approach because the converter does not really check the version, it actually checks for the expansion by looking for the listed files. The version number is just the minimum version required for this expansion. I don't even know how to solve the problem in your issue with the current converter :D Version detection by just looking for files doesn't really work well here.

I suppose it's better to leave the file here as is, and instead give an additional hint here:

https://github.com/SFTtech/openage/blob/9f13a91184e16af761fd9b654ff66cb3665261dd/openage/convert/main.py#L116

If there's a broken version, you could give a hint here that you have to downgrade to patch 4.3 and provide a link to the troubleshooting guide. It's not the nicest workaround, but since this will be replaced anyway with #1151 we could get away with it.

Reply:
@heinezen I was under the impression that the version determined the presence of the files it checked, considering that in #1217 I didn't own any DLCs yet it still detected 5.0. I'm not sure how to test below 4.6 since I can only get my hands on 4.6 and 5.0 through Steam.

Reply:
I think it detected Rajas because they added demo files of RoR, even if you don't have the expansion.

Reply:
So can we detect if there's RoR demo files and then not detect 5.0?

Reply:
@stephen304 can you give us the output of `tree` of your `Data` directory please? Then we can see what files are there and compare that to the expected layout for your version